jscomp/test/basic_module_test.js

```diff
@@ -8,14 +8,16 @@ var Mt_global = require("./mt_global.js");
 
 var count = /* record */[/* contents */0];
 
-var M = [Offset.M[/* Set */1]];
+var M = /* module */{
+  Set: Offset.M.Set
+};
 
```

Comment:
Take this for example, do you think it is good to simplify as `Offset.M`?

Reply:
Yes, looks like a low-hanging fruit. I guess it only matters on heavy runtime use of functors, so kind of niche, but if it's easy to do why not.